src/prefabs/object-menu.tsx

```diff
@@ -198,6 +199,8 @@ function ScaleButton(props: Attrs) {
       height={buttonHeight}
       type={BUTTON_TYPES.ACTION}
       text={"scale"}
+      holdable
+      holdableButton
```

Comment:
Bit weird to have to specify `holdableButton` on a `Button3D` should probably be handled in `Button3D` based on just `holdable`.

The disconnect between things being components and things just being properties to a JSX function here is definitely something I don't love, but its just how JSX works. Like here "height" is not a component nor is "text". 

I think mixing components "holdable" and "holdableButton" into it makes it even more confusing.

Reply:
Updated to remove `holdableButton` here.